src/rewriters/NestFieldOutputsRewriter.ts

```diff
@@ -76,10 +76,11 @@ class NestFieldOutputsRewriter extends Rewriter {
       ) {
         const rewrittenResponse = { ...pathResponse, ...pathResponse[this.newOutputName] };
         delete rewrittenResponse[this.newOutputName];
-        return rewrittenResponse;
+        response[key] = rewrittenResponse;
```

Comment:
nit: it would be better to return a new object rather than modifying the input params directly.  Maybe return something like: `return { ...response, [key]: rewrittenResponse }` to avoid modifying the input.